applications/new_schema.py

```diff
@@ -0,0 +1,64 @@
+import graphene
+from graphene_django.filter import DjangoFilterConnectionField
+from graphene_django.types import DjangoObjectType
+
+from resources.models import Harbor
+
+from .enums import WinterStorageMethod
+from .models import BerthApplication, HarborChoice
+
+WinterStorageMethodEnum = graphene.Enum.from_enum(WinterStorageMethod)
+
+
+class HarborType(DjangoObjectType):
+    class Meta:
+        model = Harbor
+        exclude = (
+            "location",
+            "availability_type",
+        )
+
+
+class HarborChoiceType(DjangoObjectType):
+    harbor = graphene.String()
+
+    class Meta:
+        model = HarborChoice
+        exclude = ("id", "application")
+
+    # could be resolved to servicemap_id here
+    def resolve_harbor(self, info, **kwargs):
+        if self.harbor:
+            return self.harbor.servicemap_id
+        return None
+
+
+class BerthApplicationTypeNode(DjangoObjectType):
+    boat_type = graphene.String()
+    harborchoice_set = graphene.List(HarborChoiceType)
+
+    class Meta:
+        model = BerthApplication
+        interfaces = (graphene.relay.Node,)
+        filter_fields = ["id"]
+        exclude = (
+            "chosen_harbors",
+            "berth_switch",
+        )
+
+    def resolve_boat_type(self, info, **kwargs):
+        if self.boat_type:
+            return self.boat_type.id
+        return None
+
+    def resolve_harborchoice_set(self, info, **kwargs):
+        if self.harborchoice_set.count():
+            return self.harborchoice_set.all()
+
+
+class Query:
```

Comment:
I see that there's some inconsistency where sometimes a `Query` inherits `graphene.ObjectType` and somethimes it doesn't. I don't see any reason why it shouldn't inherit the common graphene base class. Is there?